src/FakeItEasy.Extensions.ValueTask/FakeItEasy.Extensions.ValueTask.csproj

```diff
@@ -40,7 +40,6 @@
   <!-- Selectively include build output for package -->
   <ItemGroup>
     <None Include="$(OutputPath)/**/$(AssemblyName).dll" Pack="true" PackagePath="lib" Visible="false" />
-    <None Include="$(OutputPath)/**/$(AssemblyName).pdb" Pack="true" PackagePath="lib" Visible="false" />

```

Comment:
I am surprised there's no equivalent to this in FakeItEasy.csproj. Maybe the AllowedOutputExtensionsInPackageBuildOutputFolder takes its place?

Reply:
> Maybe the AllowedOutputExtensionsInPackageBuildOutputFolder takes its place?

Yes. We have to manually include the output in Extensions.ValueTask, because we don't want the netstandard2.1 output.